src/components/TeamMember/ExpandedInfo/ExpandedInfo.styles.tsx

```diff
@@ -0,0 +1,120 @@
+import styled from "styled-components";
+import Image from "next/image";
+import Link from "next/link";
+import { Icon as DefaultIcon } from "@/components/Icon/Icon";
+
+export const ExpandedInfoWrapper = styled.div`
+  display: flex;
+  border-radius: 0.8rem;
+  flex-direction: column;
+  background-color: ${({ theme }) => theme.colors.white};
+  background-image: url("/flower-background.png");
+  background-size: cover;
+  z-index: 1;
+  max-width: 35rem;
+  margin: 1rem;
```

Comment:
```suggestion
  margin: 1rem;
  padding: 1rem;
```

This padding replaces almost all the margin and padding that I suggested removing